### ISSUE TITLE
Datapack invalid

### ISSUE THREAD

**Issue by ConqAra** [MoonflowerTeam/pollen#39]
### What version are you seeing the problem on?

1.18.2

### Describe the issue

```markdown
Won't create or start up any world, keep getting "datapacks invalidated" after installing pollen in the modpack.
Tested without grind enchantments as well as I saw it was listed as "incompatible" and still had the same results.
```

### latest.log

```shell
https://pastebin.com/7XTQNLy2
```

**Comment by Magiwarriorx** [comment]
I am also getting this error on 1.18.2 Forge. Disabling Pollen resolves the issue.

**Comment by Nekkok** [comment]
Same here

**Comment by Ocelot5836** [comment]
This is the same cause as #34

**Comment by jacksonhardaway** [comment]
Fixed in Pollen 1.4.2
